js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -114,7 +114,7 @@ class RequestsPage extends Component {
         title={
           <FormattedMessage
             id='signer.requestsPage.pendingTitle'
-            defaultMessage='Pending Requests'
+            defaultMessage='Pending Transactions'
```

Comment:
Ah! I am super glad you saw these. I meant to point it out to be disputed. I feel Requests is simply somewhat misleading. We need better terminology, correct? Otherwise, I will drop it.